### PR TITLE
Focus Problem for Intro to Graphs

### PR DESCRIPTION
**Pull request by ryanchou-dev**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.

### CODE REVIEW

**Comment by ryanchou-dev** [comment]
fixes #3541

File: content/2_Bronze/Intro_Graphs.mdx [hunk 1]
```diff
@@ -71,6 +71,256 @@ categories:
 
 Knowing [DFS](/silver/dfs) can be helpful but it should not be required.
 
+## Example - Livestock Lineup
+
+<FocusProblem problem="tutorial" />
+
+This problem has multiple solutions, try to find one that runs in purely $\mathcal{O}(n)$ time.
```
Comment (SansPapyrus683) [inline content/2_Bronze/Intro_Graphs.mdx:78]: runon

File: content/2_Bronze/Intro_Graphs.mdx [hunk 2]
```diff
@@ -71,6 +71,256 @@ categories:
...
+<Spoiler title="Hint 1">
+Picture a graph with nodes numbered $1 \dots 8$, where connect two nodes if they must be milked beside each other.
```
Comment (SansPapyrus683) [inline content/2_Bronze/Intro_Graphs.mdx:80]:
```suggestion
How can we represent this problem as a graph?
```

File: content/2_Bronze/Intro_Graphs.mdx [hunk 3]
```diff
@@ -2,7 +2,7 @@
 id: intro-graphs
 title: Introduction to Graphs
 author: Darren Yao, Benjamin Qi
-contributors: Nathan Gong
+contributors: Nathan Gong, Ryan Chou, Neo Wang, Aadit Ambadkar, Sandeep Wadhwa, Kevin Sheng, Nikhil Chatterjee
```
Comment (SansPapyrus683) [inline content/2_Bronze/Intro_Graphs.mdx:5]: i'm ngl idt neo, sandeep, or aadit have anything in the final sol :skull:

Comment (ryanchou-dev) [reply content/2_Bronze/Intro_Graphs.mdx:5]: okay LMAO

File: content/2_Bronze/Intro_Graphs.mdx (hunk 1 above)
Comment (ryanchou-dev) [reply content/2_Bronze/Intro_Graphs.mdx:78]: my fault :<

**Review by envyaims — APPROVED** [review]

**Pull request merged**